packages/lesswrong/server/search/elastic/ElasticConfig.ts

```diff
@@ -221,7 +222,7 @@ const elasticSearchConfig: Record<AlgoliaIndexCollectionName, IndexConfig> = {
   },
   Users: {
     fields: [
-      "displayName^10",
+      "displayName^10000",
```

Comment:
Is this to be interpreted literally as putting 10,000 times the weight on display name? Seems like a lot

Reply:
Yeah - it's pretty weird, but just from experimentation this seemed to give me the best results. I know the scaling on the `boost` parameter isn't exactly linear, so I think it's quite likely that this isn't either.